app/src/main/java/com/swent/assos/ui/components/DeleteButton.kt

```diff
@@ -6,5 +6,5 @@ import androidx.compose.runtime.Composable
 
 @Composable
 fun DeleteButton(callBack: () -> Unit) {
-  BasicButtonWithIcon(buttonName = "Delete", callback = callBack, icon = Icons.Default.Delete)
+  BasicButtonWithIcon(buttonName = "", callback = callBack, icon = Icons.Default.Delete)
```

Comment:
Here you are deleting the string next to the icon, is the Icon explicit enough ?

Reply:
Yes it is, look at eventDetails capture

Reply:
I see thank, looks much better